content/en/guides/get-started/commands.md

```diff
@@ -61,6 +61,31 @@ You can run different commands depending on the [target](/docs/2.x/features/depl
 - **nuxt generate** - Build the application (if needed), generate every route as a HTML file and statically export to `dist/` directory (used for static hosting).
 - **nuxt start** - serve the `dist/` directory like your static hosting would do (Netlify, Vercel, Surge, etc), great for testing before deploying.
 
+## Webpack Config Inspection 
+
+You can generate the webpack config with an api heavily inspired by [vue inspect](https://cli.vuejs.org/guide/webpack.html#inspecting-the-project-s-webpack-config)
```

Comment:
- `generate` ~> `inspect`
- Neither api or cli is not inspired from vue inspect. I would suggest mentioning as similar to `vue-cli`inspect command

Reply:
Applied by https://github.com/nuxt/nuxtjs.org/commit/4597314ace8cb0a51129c65872dc12067128ecc6